ladybug_comfort/cli/map.py

```diff
@@ -516,7 +517,7 @@ def irradiance_contrib(
               'from it to get indirect irradiance.', default=True, show_default=True)
 @click.option('--output-file', '-f', help='Optional file to output the CSV matrix '
               'of MRT deltas. By default this will be printed out to stdout',
-              type=click.File('w'), default='-', show_default=True)
+              type=click.File('wb'), default='-', show_default=True)
```

Comment:
I know that I'm asking for a lot here but can we expose the option to write as binary vs. plain text as a flag option on the command? I'm thinking that this would look like the following:
```
@click.option(
    '--plain-text/--binary', ' /-b', help='Flag to note whether the output should be formatted as a '
    'plain text CSV or whether it should be formatted as a binary numpy array.')
```
This way, you don't have to update all of the tests and we won't loose the plain text capability for those who might still want it.

And we'll change the recipe by adding this flag into the pollination-ladybug-comfort function.

Reply:
I added this option to the CLI commands.